rpc/rpc.go

```diff
@@ -148,6 +149,7 @@ func NewLinkWithSession(session *amqp.Session, address string, opts ...LinkOptio
 	receiverOpts := []amqp.LinkOption{
 		amqp.LinkSourceAddress(address),
 		amqp.LinkTargetAddress(link.clientAddress),
+		amqp.LinkCredit(defaultReceiverCredits),
```

Comment:
Mmh, that's making it default for anyone constructing the a link.
I don't think that's a good approach. It must be for the RPC calls channel only

Reply:
But that's in the RPC package... I see

Reply:
Merging.